src/GraphQlSdk.php

```diff
@@ -539,6 +542,60 @@ public function refundCreate(RefundCreateInput $input) {
         return false;
     }
 
+    /**
+     * @param \Rvvup\Sdk\Inputs\GetOrderInput $input
+     * @return false|array = [
+     *     'id' => 'Rvvup Order Id',
+     *     'payments' => [
+     *         [
+     *             'id' => 'Rvvup Payment Id',
+     *             'refunds' => [
+     *                 'id' => 'Rvvup Refund Id',
+     *                 'status' => 'Rvvup Refund Status',
+     *                 'reason' => 'Rvvup Refund Reason'
+     *             ],
+     *         ],
+     *     ]
+     * ]
+     * @throws \Rvvup\Sdk\Exceptions\NetworkException
+     * @throws \JsonException
+     * @throws \Exception
+     */
+    public function getOrderRefunds(GetOrderInput $input)
```

Comment:
I am thinking for `getAnObject` requests maybe it's worth just having the just the ID in the parameters. I don't see them having complicated data objects being passed through.

```suggestion
    public function getOrderRefunds(string $orderId)
```

And we just use input objects for mutations

Reply:
fair point for complexity and follows GraphQL logic of having Input Types for mutations only.
On the other hand, I feel like having a consistent input logic (ie. interface) for any API method helps in the cognitive process of building libraries.
We shouldn't and wouldn't need to know the underlying technology of fetching results (REST, SOAP, GraphQL etc), so consistency would follow the library aspect and not the tech itself. If that makes sense?

But of course we can change it if you want.

Reply:
I think regardless of REST/GraphQL when you are getting an individual object you generally do it by an ID, so i think it's okay in that regard.

I just feel it's cleaner when you can do things like `getOrder(id)` as it's clearer when reading the code vs `getOrder(GetOrderInputFactory->create(id));` when generally speaking these cases will only ever have one field?

Reply:
@aashwin-rvvup this is now updated along with the tests